packages/theme/components/layout/sidebar/Sidebar.tsx

```diff
@@ -109,7 +115,7 @@ export function Sidebar({pageMap}: SidebarProps) {
             const shouldShowTabs = indexPage.frontMatter?.['show-tabs'] ?? false
             if (shouldShowTabs) {
               return (
-                <NavList.Item key={indexPage.name} href={`${basePath}${indexPage.route}`}>
+                <NavList.Item as={NextLink} key={indexPage.name} href={`${basePath}${indexPage.route}`}>
```

Comment:
Nice. Didn't know NavList Item supported `as`. 

Could you also remove the `basePath` in all instances please, as it should automatically apply now.

Reply:
Done! I also added `as={NextLink}` to the Breadcrumbs